package.json

```diff
@@ -67,5 +67,8 @@
   },
   "lint-staged": {
     "*.{js,ts,jsx,tsx,svelte}": "eslint --quiet --cache --fix"
+  },
+  "dependencies": {
+    "seedrandom": "^3.0.5"
```

Comment:
I think this shouldn't be a top-level dependency. Most likely you want it to be in `packages/dev`

Reply:
Good call.